Code/Framework/AzToolsFramework/AzToolsFramework/ActionManager/Menu/MenuManager.cpp

```diff
@@ -185,5 +206,60 @@ namespace AzToolsFramework
 
         return menuBarIterator->second.GetMenuBar();
     }
+    
+    MenuManagerIntegerResult MenuManager::GetSortKeyOfActionInMenu(const AZStd::string& menuIdentifier, const AZStd::string& actionIdentifier)
+    {
+        auto menuIterator = m_menus.find(menuIdentifier);
+        if (menuIterator == m_menus.end())
+        {
+            return AZ::Failure(AZStd::string::format(
+                "Menu Manager - Could not get sort key of action \"%s\" in menu \"%s\" - menu has not been registered.", actionIdentifier.c_str(), menuIdentifier.c_str()));
+        }
+
+        if (!menuIterator->second.ContainsAction(actionIdentifier))
+        {
+            return AZ::Failure(AZStd::string::format(
+                "Menu Manager - Could not get sort key of action \"%s\" in menu \"%s\" - action was not found in menu.", actionIdentifier.c_str(), menuIdentifier.c_str()));
+        }
+
+        return AZ::Success(menuIterator->second.GetActionSortKey(actionIdentifier));
+    }
+
+    MenuManagerIntegerResult MenuManager::GetSortKeyOfSubMenuInMenu(const AZStd::string& menuIdentifier, const AZStd::string& subMenuIdentifier)
```

Comment:
update: This looks like it could also be `const`?

Reply:
Fixed